Listener.js

```diff
@@ -66,11 +66,30 @@ class Listener {
 
     console.log(`Listening to subscription: ${subscriptionFullName}`);
 
-    const onMessage = (msg) => {
-      onMessageCallback(msg);
-      msg.ack();
+    const onMessageTryCatch = (msg) => {
+      try {
+        onMessageCallback(msg);
```

Comment:
note the try catch behaviour only  works with onMessageCallback being a synchronous method. If it was a promise or standard callback function, the try catch would not be triggered